xzes40/transformer/src/lib.cpp

```diff
@@ -170,8 +161,18 @@ bool xzes::_file_exists( std::string file_path )
 // This is hopefully a temporary measure and will eventually become a wrapper
 // for a *real* hash function.
 // ----------------------------------------------------------------------------
-std::string xzes::_hash( std::string input )
+int xzes::_hash( const char* s )
```

Comment:
This hash function hash the string to integer. It is easy function, but less collision.